tdrs-backend/tdpservice/parsers/schema_defs/tanf/t3.py

```diff
@@ -0,0 +1,117 @@
+        ]),
+        Field(name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8, required=True, validators=[
+        ]),
+        Field(name='CASE_NUMBER', type='string', startIndex=8, endIndex=19, required=True, validators=[
+        ]),
+        Field(name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20, required=True, validators=[
+        ]),
+        Field(name='DOB', type='number', startIndex=20, endIndex=28, required=True, validators=[
+        ]),
+        Field(name='SSN', type='string', startIndex=28, endIndex=37, required=True, validators=[
+        ]),
+        Field(name='RACE_HISPANIC_OR_LATINO', type='string', startIndex=37, endIndex=38, required=True, validators=[
+        ]),
+        Field(name='RACE_AMERICAN_INDIAN_OR_ALASKA_NATIVE', type='string', startIndex=38, endIndex=39, required=True,
+              validators=[]),
+        Field(name='RACE_ASIAN', type='string', startIndex=39, endIndex=40, required=True, validators=[
+        ]),
+        Field(name='RACE_BLACK_OR_AFRICAN_AMERICAN', type='string', startIndex=40, endIndex=41, required=True,
+              validators=[]),
+        Field(name='RACE_NATIVE_HAWAIIAN_OR_OTHER_PACIFIC_ISLANDER', type='string', startIndex=41, endIndex=42,
+              required=True, validators=[]),
+        Field(name='RACE_WHITE', type='string', startIndex=42, endIndex=43, required=True, validators=[
+        ]),
+        Field(name='GENDER', type='number', startIndex=43, endIndex=44, required=True, validators=[
+        ]),
+        Field(name='RECEIVES_BENEFITS_UNDER_NON_SSA_PROGRAMS', type='string',
+              startIndex=44, endIndex=45, required=True, validators=[]),
+        Field(name='RECEIVES_SSI_UNDER_TITLE_XVI_SSI', type='string', startIndex=45, endIndex=46, required=True,
+              validators=[]),
+        Field(name='RELATIONSHIP_TO_HEAD_OF_HOUSEHOLD', type='number', startIndex=46, endIndex=48, required=True,
+              validators=[]),
+        Field(name='PARENT_WITH_MINOR_CHILD', type='string', startIndex=48, endIndex=49, required=True, validators=[
+        ]),
+        Field(name='EDUCATION_LEVEL', type='string', startIndex=49, endIndex=51, required=True, validators=[
+        ]),
+        Field(name='CITIZENSHIP_ALIENAGE', type='string', startIndex=51, endIndex=52, required=True, validators=[
+        ]),
+        Field(name='UNEARNED_INCOME_SSI', type='string', startIndex=52, endIndex=56, required=False, validators=[]),
+        Field(name='UNEARNED_INCOME_OTHER_UNEARNED_INCOME', type='string', startIndex=56, endIndex=60, required=False,
+              validators=[]),
+    ],
+)
+
+child_two = RowSchema(
+    model=TANF_T3,
+    quiet_preparser_errors=True,
+    preparsing_validators=[
+        validators.notEmpty(start=60, end=101),
+    ],
+    postparsing_validators=[],
+    fields=[
+        Field(name='RecordType', type='string', startIndex=0, endIndex=2, required=True, validators=[
+        ]),
```

Comment:
Nitpick: Not sure if this is a side-effect of linting but we have inconsistent newlines between the brackets of `validators=[]`. Would prefer this to be consistent with how we're going about it in #2479 which doesn't split the line but looks like this:

```
   fields=[
        Field(name='RecordType', type='string', startIndex=0, endIndex=2,
              required=True, validators=[]),
        Field(name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
              required=True, validators=[]),
             ```